docs/modules/wan/pages/wan.adoc

```diff
@@ -50,4 +50,4 @@ clusters or multiple external systems.
 identifier can then be used to control the behavior of a WAN replication publisher while the source/active cluster is
 running. For instance, you can use this identifier in combination with the WAN replication scheme name to pause, stop or
 resume WAN replication for that specific publisher. Or, you can trigger synchronization with a specific target cluster
-and so on.
\ No newline at end of file
+and so on.
```

Comment:
I'd suggest recasting the previous two sentences (the second is really only a phrase and the last three words that were deleted and re-added don't make sense in it); something like:

For example, you could use the WAN Publisher ID as follows: 
+
* In combination with the WAN replication scheme name to pause, stop, or resume WAN replication for a specific publisher
* To trigger synchronization with a specific target cluster